src/saveload/oldloader_sl.cpp

```diff
@@ -416,7 +416,7 @@ static bool FixTTOEngines()
 			if (TimerGameCalendar::date >= e->intro_date && HasBit(e->info.climates, 0)) {
 				e->flags |= ENGINE_AVAILABLE;
 				e->company_avail = MAX_UVALUE(CompanyMask);
-				e->age = TimerGameCalendar::date > e->intro_date ? (TimerGameCalendar::date - e->intro_date) / 30 : 0;
+				e->age = TimerGameCalendar::date > e->intro_date ? (int32_t)(TimerGameCalendar::date - e->intro_date) / 30 : 0;
```

Comment:
We kinda need DateToMonth or something here too. The `/ 30` is unclear as it is.

Reply:
Not for this PR. It needs a cast either way, so this is fine.